src/traces/violin/attributes.js

```diff
@@ -155,6 +155,7 @@ module.exports = {
     hovertemplate: boxAttrs.hovertemplate,
 
     quartilemethod: boxAttrs.quartilemethod,
+    showwhiskers: boxAttrs.showwhiskers,
```

Comment:
Not clear to me that we really want to add `showwhiskers` to violins, but if we do, they should be defined inside the `box` container below (and then whatever is needed for that setting to make it to the drawing routine) with `dflt: true` and their own description.

Anyway I'd be fine with omitting this for now, in which case the condition during draw can be `if (showwhiskers !== false)` instead of `if (showwhiskers)` so that `undefined` behaves like `true`.